script.js

```diff
@@ -17,3 +20,133 @@ mobNavItems.forEach((mobNavItem) => {
+
+  const modalContainer = document.createElement('div');
+  modalContainer.classList.add('modal-container');
+  const modal = document.createElement('div');
+  modal.classList.add('modal');
+  modalContainer.append(modal);
+  worksSection.append(modalContainer);
+  modalContainer.setAttribute('id', 'modal-container');
+
+  Array.prototype.slice.call(projectCards).forEach((eachCard, index) => {
+    const cardBtn = eachCard.childNodes[5].childNodes[9];
+
+    cardBtn.addEventListener('click', () => {
+      document.body.classList.add('stop-scrolling');
+      modalContainer.style.transform = 'translateX(0)';
+      modal.innerHTML = `
+      <img id="close-modal" src="./design/assets/close-modal.svg" />
+      <h2 class="project-name">${projectNames[index]}</h2>
+      <div class="project-details">
+        <h3>${projectInfo[index][0]}</h3>
+        <img src="./design/assets/Counter.svg" alt="Decorative separator dot image">
+        <h3>${projectInfo[index][1]}</h3>
+        <img src="./design/assets/Counter.svg" alt="Decorative separator dot image">
+        <h3>${projectInfo[index][2]}</h3> 
+      </div>
+      <img class="modal-desktop-img" src="${desktopImage[index]}" />
+      <div class="modal-details">
+        <p>${projectDescriptions}</p>
+        <div>
+        <ul class="project-used-techs modal-used-techs">
+          <li>${projectSkills[index][0]}</li>
+          <li>${projectSkills[index][1]}</li>
+          <li>${projectSkills[index][2]}</li>
+        </ul>
+        <div class="see-more-btns">
+          <a onclick="window.open('https://z4cope.github.io/Mobile-portfolio/')" target="_blank" class="call-to-action-btns modal-btn">See live <img src="./design/assets/see-more.svg" /></a>
+          <a onclick="window.open('https://github.com/z4cope/Mobile-portfolio')" target="_blank" class="call-to-action-btns modal-btn">See Source <img src="./design/assets/github-link.svg" /></a>
+        </div>
+        </div>
+      </div>
+      `;
+      const closeBtn = document.getElementById('close-modal');
+      closeBtn.addEventListener('click', () => {
+        modalContainer.style.transform = 'translateX(-100%)';
+        document.body.classList.remove('stop-scrolling');
+      });
+    });
+  });
+});
+
+const userForm = document.getElementsByTagName('form')[0];
+const userEmail = document.getElementById('user-email');
+let validValue = false;
+const messageContainer = document.getElementById('message-box');
+
+userEmail.addEventListener('input', (e) => {
+  const validEmailValue = e.target.value.toLowerCase();
+  if (validEmailValue === e.target.value.toLowerCase()) {
+    validValue = true;
+  } else {
+    validValue = false;
+  }
+});
+
+userForm.addEventListener('submit', (e) => {
+  if (!validValue) {
+    const message = document.createElement('p');
+    message.innerText = 'Please fill your email properly';
+    messageContainer.append(message);
+    messageContainer.style.opacity = '1';
+    e.preventDefault();
+  }
```

Comment:
- [ ] Good job so far 👏  However, the form could still be submitted while it is not supposed to submit. Please try to go through your algorithm again to see what's wrong. Normally, the form should not be submitted when the Email is in capital letters 😃

Reply:
@BertrandConxy, Thank you so much for your comment, but the form is submitted because I took the uppercased value and turned it to lowercase before it's even submitted, you can kindly test that by running a console.log() on the email value to the results. Thank you again.